### PR TITLE
Automated cherry pick of #118356: Add ephemeralcontainer to imagepolicy securityaccount

### PR DESCRIPTION
**Pull request by ritazh**
Cherry pick of #118356 on release-1.24.

#118356: Add ephemeralcontainer to imagepolicy securityaccount

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

```release-note
NONE
```
/kind bug
/sig auth
/milestone v1.24

### CODE REVIEW

**Comment by jeremyrickard** [comment]
@deads2k  this one needs an approval from you before we can approve. I approved the other branch Cps

**Review by xmudrii — APPROVED** [review]
We'll have at least one more 1.24 release. For RelEng:
/lgtm
/approve
/hold cancel

**Pull request merged**
